tests/phpunit/tests/plugins/test-duplicate-post.php

```diff
@@ -6,16 +6,13 @@ class Duplicate_Post_Test extends PLL_UnitTestCase {
 	 * @param WP_UnitTest_Factory $factory
 	 */
 	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
-
-		if ( ! file_exists( DIR_TESTROOT . '/../duplicate-post/' ) ) {
-			self::markTestSkipped( 'This test requires the plugin Duplicate Post' );
-		}
-
-		require_once DIR_TESTROOT . '/../duplicate-post/duplicate-post.php';
-		require_once DIR_TESTROOT . '/../duplicate-post/admin-functions.php';
+		self::assertFileExists( PLL_TEST_EXT_PLUGINS_DIR . 'duplicate-post/duplicate-post.php', 'This test requires the plugin Duplicate Post.' );
```

Comment:
Sorry I missed your question on Slack about that yesterday.
I must say I'm not very fond of.
Reason being I don't always install plugins/themes for integration tests locally. I don't think it's useful to run compatibility tests each time I modify code on my machine and it takes quite some times to run theses tests...
And with that code, the suite will fail every time.
Could we skip the tests gracefully instead?
It'll still be clear in CI whether or not compatibility tests run or not.

Reply:
I will be clear only if we go see the results of each action, because everything will be green.
However, we could add tags and create composer scripts to run tests without the plugins. But I think it should be in another PR.

Reply:
Then we should implement our own `markTestSkipped` that fails only in CI.

Reply:
Something like `markTestSkippedOrDieInCI`, so it rhymes.

Reply:
Done in [26e0b06](https://github.com/polylang/polylang/pull/1581/commits/26e0b0635aaa25bec11bcedd1bf37cd8ffa83e73).
An env var `PLL_SKIP_PLUGINS_TESTS` must be set in order to skip instead of asserting.